modules/nw-enforcing-hsts-per-domain.adoc

```diff
@@ -47,19 +52,19 @@ metadata:
   name: cluster
 spec:
   domain: 'hello-openshift-default.apps.username.devcluster.openshift.com'
-  requiredHSTSPolicies: <1>
-  - domainPatterns: <2>
+  requiredHSTSPolicies: # <1>
```

Comment:
This is part of the ingress resource in openshift config apigroup, MicroShift does not have it. Do we need `ifndef` here too?